test/pynrn/test_basic.py

```diff
@@ -190,3 +192,7 @@ def test_HocObject_no_deferred_unref():
     print(sec)
     sl.append(sec=sec)
   assert(len([s for s in sl]) == 0)
+
```

Comment:
I assume this part was for local testing.

Reply:
Yep.